content/articles/mock-a-datasource-for-react-using-json-server/index.md

```diff
@@ -0,0 +1,837 @@
+When building a React project, developers need some quick-to-use data sources with endpoints. These data sources provide test data to test whether the package works correctly.
+Datasource mainly used may include DBMS systems such as MySQL, Postgres, SQL, among many others. The problem with this approach is that during prototyping, e.g., in Rapid Prototyping Approach, it will be quickly discarded if the application prototype is not as expected.
+The extra setup and configurations will be tedious and useless since they will be quickly discarded.
+
+The process produces a need to create a data source that can be quickly set up and discarded, yet it performs the same functionality as the DBMS.
+It should also be small enough for fewer storage requirements and produce correct and accurate results as expected.
+The use of the Json-Server library can solve such data sources problems.
```

Comment:
json-server is the name of the library?

If so, make it `json-server` ( enclose within ` `` `)